packages/twenty-front/src/modules/ui/utilities/page-favicon/components/PageFavicon.tsx

```diff
@@ -16,7 +17,7 @@ export const PageFavicon = () => {
             getImageAbsoluteURI({
               imageUrl: workspacePublicData.logo,
               baseUrl: REACT_APP_SERVER_BASE_URL,
-            }) ?? ''
+            }) ?? DEFAULT_WORKSPACE_LOGO
```

Comment:
@Bonapara FYI, I'm also making sure we have a Favicon (we cannot use the first letter trick here as the logo needs to be a png)